client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/RemoteOffHeapQueryDslConditionsTest.java

```diff
@@ -0,0 +1,24 @@
+package org.infinispan.client.hotrod.query;
+
+import static org.infinispan.server.hotrod.test.HotRodTestingUtil.hotRodCacheConfiguration;
+
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.Index;
+import org.infinispan.configuration.cache.StorageType;
+import org.testng.annotations.Test;
+
+/**
+ * @author anistor@redhat.com
+ */
+@Test(groups = "functional", testName = "client.hotrod.query.RemoteOffHeapQueryDslConditionsTest")
+public class RemoteOffHeapQueryDslConditionsTest extends RemoteQueryDslConditionsTest {
+
+   protected ConfigurationBuilder getConfigurationBuilder() {
```

Comment:
This method would be nicer if based on ```super```.

```
   protected ConfigurationBuilder getConfigurationBuilder() {
      ConfigurationBuilder builder = super.getConfigurationBuilder();
      builder.memory().storageType(StorageType.OFF_HEAP);
      return builder;
   }```

Reply:
changed